bulbs/poll/models.py

```diff
@@ -0,0 +1,94 @@
+from bulbs.content.models import Content
+from bulbs.utils import vault
+from django.db import models
+from djes.models import Indexable
+from rest_framework.exceptions import APIException
+
+from time import mktime
+import pytz
+import requests
+
+SODAHEAD_DATE_FORMAT = '%m/%d/%y %I:%M %p'
+
+class Poll(Content):
+    class SodaheadResponseError(APIException):
```

Comment:
Usually, (qualifier) from my experiences, nested classes should be reserved for namespace conflicts as all it does is limit reusability.